scripts/generator/graphql_generator/csharp/SDK/Android/NativeMessage.Android.cs.erb

```diff
@@ -0,0 +1,24 @@
+#if UNITY_ANDROID
+namespace <%= namespace %>.SDK {
+    using System;
+    using System.Collections.Generic;
+    using <%= namespace %>.MiniJSON;
+
+    #if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    #endif
+
+    public partial class NativeMessage {
+        private AndroidJavaObject MessageCenter;
```

Comment:
Should this be static? Otherwise we would create a new MessageCenter reference for each message